stdlib/LinearAlgebra/test/svd.jl

```diff
@@ -139,4 +139,35 @@ aimg  = randn(n,n)/2
     end
 end
 
+
+
+@testset "SVD Algorithms" begin
+    ≊(x,y) = isapprox(x,y,rtol=1e-15)
```

Comment:
I think you can use plain vanilla
```julia
x ≈ y rtol = 8*eps()
```
in tests below.

Reply:
Doesn't really matter, right? Also, I'd have to repeat the `rtol = 8*eps()` a bunch of times.

Reply:
Also note that I copied this idea from further up in the file (a piece not added by me).